pkgs/applications/science/electronics/simulide/default.nix

```
@@ -0,0 +1,79 @@
+{ lib
+, stdenv
+, fetchbzr
+, libsForQt5
+}:
+
+let
+  version = "0.4.15";
+  release = "SR10";
+  branch = "simulide_0.4.14"; # the branch name does not mach the version for some reason
+  rev = "291";
+  sha256 = "sha256-BBoZr/S2pif0Jft5wrem8y00dXl08jq3kFiIUtOr3LM=";
+in
+stdenv.mkDerivation {
```

This will use the qt's custom `mkDerivation`
```suggestion
mkDerivation {
```